apps/frontend/src/libs/components/route-card/route-card.tsx

```diff
@@ -0,0 +1,37 @@
+import StarIcon from "./star.svg?react";
+import styles from "./styles.module.css";
+
+type Properties = {
+	alt: string;
+	image?: string;
+	label: string;
```

Comment:
let's make it closer to how we will receive it from the db
```suggestion
	imageUrl: string | null;
	name: string;
```